kura/distrib/src/main/sh/extract.sh

```diff
@@ -57,6 +57,7 @@ function run_kura_install {
             if [ "$DELTA" -ge "$TIMEOUT_TIME" ]; then
                 echo "The installation process is not responding. It'll be stopped."
                 kill -9 $PID >> /dev/null 2>&1
+                exit 2
```

Comment:
Is it better to add the `stop_watchdog` call also before the `exit 2` command?

Reply:
I am not sure. I guess when the script terminates, the watchdog would end too.

Reply:
I don't think so. If not stopped or continuously kicked, the watchdog eventually reboot the device.

Reply:
Isn't that a good idea in this case?

Reply:
Not sure... a reboot would not solve the error.